schema/nanotdf/README.md

```diff
@@ -343,21 +347,31 @@ the Resource Locator.
 ##### 3.4.1.3 Body
 
 The data required to retrieve the Resource referenced by the Resource Locator.
-The type of the data contained in the Body is determined by the [Protocol Enum]. For `http` or `https` the data contained in this
-section would be everything following the `://` in a URL.
+The type of the data contained in the Body is determined by the [Protocol Enum].
+For `http` or `https` the data contained in this section would be everything following
+the `://` in a URL.
 
 #### 3.4.2 Policy
 
 [Policy]: #342-policy
 [Section 3.4.2]: #342-policy
 
-The structure of the Policy is as follows:
+The structure of the Remote Policy is as follows:
+
+| Section   | Minimum Length (B) | Maximum Length (B) |
+| --------- | ------------------ | ------------------ |
+| Type Enum | 1                  | 1                  |
+| Body      | 3                  | 257                |
+| Binding   | 8                  | 132                |
+
+The structure of the Embedded Policy is as follows:
 
-| Section       | Minimum Length (B)  | Maximum Length (B)  |
-|---------------|---------------------|---------------------|
-| Type Enum     | 1                   | 1                   |
-| Body          | 3                   | 257                 |
-| Binding       | 8                   | 132                 |
+| Section       | Minimum Length (B) | Maximum Length (B) |
+| ------------- | ------------------ | ------------------ |
+| Type Enum     | 1                  | 1                  |
+| Policy Length | 2                  | 2                  |
+| Body          | 1                  | 64,000             |
```

Comment:
My recollection is body should be 3 - 257 bytes as in the original spec. I *think* the diagrams are wrong. I'm definitely open to loosening the restriction here since we no longer are bound by the same requirements as we were when the spec was originally written.